src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+    let totalCost = days * dailyRate;
+
+    if (days >= 7) {
+        totalCost -= 50;
+    } else if (days >= 3) {
+        totalCost -= 20;
```

Comment:
Similarly, avoid using literals for the '3' and '20' values. Define constants such as 'SHORT_TERM_RENTAL_DAYS' and 'SHORT_TERM_DISCOUNT' to describe these values.